### DISCUSSION TITLE
Bootstrap 6 release date

### DISCUSSION THREAD

**Discussion by roshanpw** [twbs/bootstrap#40572] [Q&A]
Hi,
Anybody know when will Bootstrap 6 release date? or when will be ended of life Bootstrap 5

**Comment by julien-deramond** [comment]
Hi @roshanpw,
Unfortunately, we can't provide a specific date at this time. We're currently focused on finalizing patch versions of Bootstrap 5 and have several features to integrate before starting on v6. So, it won't be soon, I'd say.

**Reply by superzackx** [reply]
Any updates on the release?

**Reply by julien-deramond** [reply]
Unfortunately not, we're still working on v5.3.x and v5.x versions right now.

**Comment by coliff** [comment]
heads-up @roshanpw @superzackx - in case you hadn't seen - there's an early work-in-progress branch for v6 now!
https://github.com/twbs/bootstrap/pull/41236

**Comment by richware** [comment]
I am currently migrating version 5.3.6 from @imports to @use/@forward. Currently using JetBrain IDE for sass and they have migrated to Dart Sass 1.89.2 which deprecates @import;

**Reply by mrholek** [reply]
@richware I did it several weeks ago in our Bootstrap fork; maybe it will be helpful for you: https://github.com/coreui/coreui

**Comment by peminator** [comment]
and any note on what will be the main difference in 6? 
moving to new compatibility level? drop some old?
hope to read bout that first, to see what new feats browser are supporting wide and i missed.

Imho, few kb more, im for keeping whats necesary, but some lower level importance  maybe could go leveraged in compatibility levels - users compiling own scss may have a backward-compatibility.scss maybe, to simply comment out all related includes on one place? like all the -ms, -o, -webkit,,, prefixes. Some matter more, some less. Someone wants more compat, someone does not give a flying f*ck bout that.

**Comment by Nayte91** [comment]
Is it a Santa list post?

- Nested CSS all around,
- Web components for JS components,
- All around use of show(), showModal(), anchor and popover APIs.

Bootstrap 5 will be 5yo soon. That's a lot, but as JS and CSS evolved a lot those few years, that's a good news that BS takes time to swallow new features and implements seamlessly. Good luck! ❤️
